### ISSUE TITLE
Error in images slot of bcScore function

### ISSUE THREAD

**Issue by Silve-Ruano** [cnio-bu/beyondcell#156]
I have an error when using the bcScore function, and it is that the Large Seurat Object "sc" does not have the slot images, while in the ST_analysis, the LargeSeuratObject "st" does after performing the same data processing. Thank you very much.
![image](https://github.com/cnio-bu/beyondcell/assets/157005665/0d0fcf23-e299-469c-800c-969a1102ccca)

**Comment by SGMartin** [comment]
@mj-jimenez

**Comment by SGMartin** [comment]
Hey @Silve-Ruano   sorry for the late reply

Are you trying to analyze spatial transcriptomics data or a regular sc dataset?

**Comment by Silve-Ruano** [comment]
I am trying to reproduce your paper, with the single cell data of authors Ho, Kinker, Jerby-Arnon and Stewart. I already solved the problem and the bug was in the versions of the Seurat and SeuratObject packages. Thank you very much for your reply!

**Comment by SGMartin** [comment]
@Silve-Ruano 
Yeah, we should have pinned both Seurat and SeuratObject versions but oh well. Glad it helped!
